### DISCUSSION TITLE
Warning about sending multiple emails from a single method

### DISCUSSION THREAD

**Discussion by DWalkit** [lukencode/FluentEmail#273] [General]
Hello!

Thank you for FluentEmail and the MailGun sender, they've been a fabulous asset for me and my projects.  

Yesterday, we realized we had an issue with seemingly random emails combining the "To:" addresses, "Tags", etc.  This was rather embarrassing and could've been a real privacy issue, in some cases.  

After digging through the repository "Issues" I found a person that ran into the same problem and you mentioned that you need to use IFluentEmailFactory and its Create() method versus the normal IFluentEmail.

I really think you should add a warning about this in the ReadMe, this could cause some serious issues for many and should be front and center for all users to see.  

Thanks!

David

**Comment by gdoten** [comment]
Seems like the Email class could use a Reset method to clear out all the properties, which might make an Email instance reusable for multiple emails. I tried `email.Data = new Data()` and things like `email.Data.ReplyToAddresses.Clear()` without luck, plus that relies on knowledge of the Email class' implementation.
